TelemetryAPI/SimpleEvent.cs

```diff
@@ -8,6 +8,82 @@
 namespace TelemetryAPI
 {
     public class SimpleEvent
+    {
```

Comment:
I don't think it's necessarily correct to change the original event definition and rename the old version to V1.  If you do name it V1, then this one should be V2.

Also, it looks like you only added a couple of fields, so you could probably just extend the original event with the new fields.  Schemas like this are fine with extending, its removing fields that can be an issue.

Reply:
Resolved in latest commit.

Reply:
fixed.